src/connector/src/sink/mysql.rs

```diff
@@ -163,7 +183,15 @@ async fn write_to_mysql<'a>(
         chunk
             .columns()
             .iter()
-            .map(|x| MySQLValue::try_from(x.array_ref().datum_at(idx)))
+            .zip_eq(schema.names().iter())
+            .filter_map(|(c, n)| {
+                // Exclude row id column.
+                if !n.ends_with("_row_id") {
```

Comment:
I'm afraid this is too ad-hoc for sink. It would be better to add a projection before sink in frontend plan to remove this column if you don't want it.